src/Symfony/Bundle/FrameworkBundle/Console/Application.php

```diff
@@ -70,12 +73,28 @@ public function doRun(InputInterface $input, OutputInterface $output)
 
         $this->setDispatcher($this->kernel->getContainer()->get('event_dispatcher'));
 
+        if ($this->registrationErrors) {
+            $this->renderRegistrationErrors($output);
```

Comment:
Right. Still not sure whats populating registrationErrors :S id expected it to be set after parent::doRun. 

https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Console/Application.php#L216 being the trigger